_posts/documentation/learn/2000-01-01-quick-start.md

```diff
@@ -7,6 +7,16 @@ permalink: quick-start.html
 
 This instruction assumes that PhantomJS is installed and its executable is placed somewhere in the PATH.
 
+Mac PATH suggested setup:
+  1. Open Terminal.
+  2. Type `vi ~/.bash_profile` and hit enter (this opens or creates your bash_profile where you can customize Terminal commands).
+  3. Press `i` to enter insert/edit mode then type `alias phantomjs='~/PATH/TO/phantomjs'` and be sure to replace `~/PATH/TO/phantomjs` with something like `~/Documents/PhantomJS/bin/phantomsj` or wherever the file exists on your machine.
```

Comment:
Better command, also is a one-liner: `echo "export PATH=$PATH:/location/of/phantomjs" >> .bash_profile`